packages/ndla-ui/src/LicenseByline/LicenseByline.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { getLicenseByAbbreviation } from "@ndla/licenses";
+import LicenseLink from "./LicenseLink";
+
+interface Props {
+  license: LicenseType;
+}
+export type LicenseType = ReturnType<typeof getLicenseByAbbreviation>;
+
+const LicenseByline = ({ license }: Props) => {
```

Comment:
No er dette kun en wrapper rundt LicenseLink, og brukes kun i AudioSearchResult og VideoResultList. 
Kanskje vi rett og slett skulle inlina LicenseLink desse to plassene for å omgå heile problemet!